### ISSUE TITLE
Get Notifications in Taskbar, not just system tray

### ISSUE THREAD

**Issue by seaus66** [ramboxapp/community-edition#2674]
You used to be able to see a notification of new messages in the task bar. Now you only see it in the system tray. Easy to get lost.

How do I make sure I get a counter or notification in the taskbar on the icon/logo?

### ENVIRONMENT

> Rambox: v0.7.5 (latest)
> OS: Win 10
> Arch: x64

**Comment by fvulich** [comment]
In order to get the badge in the task bar you must have this checked in the app setting
![image](https://user-images.githubusercontent.com/46904390/80000802-fbee9f80-8493-11ea-8636-7d39cf0b62a1.png)

**Comment by seaus66** [comment]
yeah, funny thing, this is checked on EVERY service, however, no counter appears. Ever.

![image](https://user-images.githubusercontent.com/48702714/80002685-24a18400-84ea-11ea-9a36-b10b0e9e6413.png)

**Comment by fvulich** [comment]
I could reproduce your issue, we'll be fixing it in our next version.
